web_widget_dropdown_dynamic/models/ir_filters.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class IrFilters(models.Model):
+    _inherit = "ir.filters"
+
+    dynamic_dropdown_int = fields.Integer("Dropdown Integer")
```

Comment:
Doesn't this create a presistent column which is useless outside its demo purposes?

Reply:
Hmm yes... any other option to do this?

Reply:
You can load it as data. Something like this should work, I think

```xml
  <record id="demo_web_widget_dropdown_dynamic" model="ir.model.fields">
    <field name="model">ir.filters</field>
    <field name="model_id" ref="base.model_ir_filters"/>
    <field name="name">dynamic_dropdown_int</field>
    <field name="selectable" eval="True"/>
    <field name="selection">[('one', 'one'), ('two', 'two')]</field>
    <field name="ttype">selection</field>
  </record>
```